contrib/tealdeer/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "tealdeer"
+pkgver = "1.6.1"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo"]
+makedepends = ["rust-std"]
+pkgdesc = "Very fast implementation of tldr"
+maintainer = "ttyyls <contact@behri.org>"
+license = "MIT"
+url = "https://github.com/dbrgn/tealdeer"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "d42db25a56a72faec173c86192656c5381281dc197171f385fccffd518930430"
+# not working
```

Comment:
should say why

Reply:
the last 8 tests required connectivity which is disabled at the check stage, I adjusted the comment accordingly

Reply:
is it possible (without involving something overly complicated) to exclude the network-using tests and let the others run?

Reply:
How about omitting the network connectivity tests by simply removing the `#[test]` annotation from them ?

Reply:
yes that looks good